stdlib/3/optparse.pyi

```diff
@@ -0,0 +1,248 @@
+def check_choice(option: Option, opt, value) -> Any: ...
+def isbasestring(x) -> bool: ...
+
+class OptParseError(Exception):
+    msg = ...  # type: Any
+    def __init__(self, msg) -> None: ...
+
+class BadOptionError(OptParseError):
+    opt_str = ...  # type: Any
+    def __init__(self, opt_str) -> None: ...
+
+class AmbiguousOptionError(BadOptionError):
+    possibilities = ...  # type: Any
+    def __init__(self, opt_str, possibilities) -> None: ...
+
+class OptionError(OptParseError):
+    msg = ...  # type: Any
+    option_id = ...  # type: str
+    def __init__(self, msg, option: Option) -> None: ...
+
+class OptionConflictError(OptionError): ...
+
+class HelpFormatter:
+    NO_DEFAULT_VALUE = ...  # type: str
+    _long_opt_fmt = ...  # type: str
+    _short_opt_fmt = ...  # type: str
+    current_indent = ...  # type: int
+    default_tag = ...  # type: str
+    help_position = ...  # type: Any
+    help_width = ...  # type: Any
+    indent_increment = ...  # type: Any
+    level = ...  # type: int
+    max_help_position = ...  # type: int
+    option_strings = ...  # type: Dict[Option, str]
+    parser = ...  # type: Any
+    short_first = ...  # type: Any
+    width = ...  # type: Any
+    def __init__(self, indent_increment, max_help_position, width, short_first) -> None: ...
+    def _format_str(self, str: str) -> str: ...
+    def dedent(self) -> None: ...
+    def expand_default(self, option: Option) -> str: ...
+    def format_description(self, description) -> Any: ...
+    def format_epilog(self, epilog) -> Any: ...
+    def format_heading(self, heading) -> Any: ...
+    def format_option(self, option: Any) -> str: ...
+    def format_option_strings(self, option: Any) -> Any: ...
+    def format_usage(self, usage) -> Any: ...
+    def indent(self) -> None: ...
+    def set_long_opt_delimiter(self, delim) -> None: ...
+    def set_parser(self, parser) -> None: ...
+    def set_short_opt_delimiter(self, delim) -> None: ...
+    def store_option_strings(self, parser) -> None: ...
+
+class IndentedHelpFormatter(HelpFormatter):
+    _long_opt_fmt = ...  # type: str
+    _short_opt_fmt = ...  # type: str
+    current_indent = ...  # type: int
+    default_tag = ...  # type: str
+    help_position = ...  # type: int
+    help_width = ...  # type: Optional[int]
+    indent_increment = ...  # type: Any
+    level = ...  # type: int
+    max_help_position = ...  # type: int
+    option_strings = ...  # type: Dict[Any, Any]
+    parser = ...  # type: Optional[OptionParser]
+    short_first = ...  # type: Any
+    width = ...  # type: Any
+    def __init__(self, *args, **kwargs) -> None: ...
+    def format_heading(self, heading) -> str: ...
+    def format_usage(self, usage) -> str: ...
+
+class Option:
+    ACTIONS = ...  # type: Tuple[str, ...]
+    ALWAYS_TYPED_ACTIONS = ...  # type: Tuple[str, ...]
+    ATTRS = ...  # type: List[str]
+    CHECK_METHODS = ...  # type: Union[None, List[Callable]]
```

Comment:
This actually can't be None; it's set to a list later in the class body. All the callables are `Callable[[Option], None]`.